public/ge0.html

```diff
@@ -20,97 +20,207 @@
 
   <link rel="stylesheet" href="https://unpkg.com/leaflet@1.7.1/dist/leaflet.css" integrity="sha512-xodZBNTC5n17Xt2atTPuE1HxjVMSvLVW9ocqUKLsCC5CXdbqCmblAshOMAS6/keqq/sMZMZ19scR4PsZChSR7A==" crossorigin="" />
   <style>
+
     html,
     body {
       margin: 0;
       padding: 0;
       width: 100%;
       height: 100%;
+      font-family: sans-serif;
+      font-weight: lighter;
+      line-height: 1.2em;
     }
-
+    
     .link {
-      background-color: white;
-      color: #00AA00;
+      border-radius: 25px;
+      border: 1px solid #449f33;
+      margin-top: 2px;
+      margin-bottom: 2px;
+      display: inline-block;
+    }
+    
+    .linkcolour {
+      border-radius: 25px;
       padding: 5px;
+      padding-left: 10px;
+      padding-right: 10px;
       text-decoration: none;
+      color: #449f33;
+      vertical-align: middle;
       display: inline-block;
-      border: 1px solid #00AA00;
     }
-
-    .link:hover {
-      background-color: #00AA00;
-      color: white;
-      cursor: grab;
+    
+    .linkcolour:hover {
+      background-color: #449f33;
+      color: #FFF;
+      cursor: pointer;
     }
-
-    a,
-    a:link,
-    a:visited {}
-
-    a:hover,
-    a:active {}
-
-    #name {
-      font-size: x-large;
-      font-weight: bold;
-      margin: 0 5px 0 5px;
+    
+    .linkcolour:active {
+      background-color: #24541b;
     }
 
-    #header {
-      color: white;
-      background-color: #00AA00AA;
+    #container {
+      background-color: #FFF;
       text-align: center;
-      position: fixed;
-      width: 100%;
-      z-index: 1000;
       padding: 10px;
+      box-shadow: 0 3px 10px 0 #00000030;
+      display: flex;
+      flex-flow: row wrap;
+      align-items: center;
+      gap: 10px;
+      justify-content: space-around;
+      overflow-wrap: break-word;
+    }
+    
+    #header {
+      z-index: 1000;
+      width: 100%;
+      position: fixed;      
+    }
+    
+    #logo {
+      vertical-align: middle; 
+      height: 2em;
+      flex-flow: row wrap;
+      align-items: center;
+      gap: 10px;
+      display: inline-flex;
+      justify-content: space-between;
+      overflow-wrap: break-word;
+    }
+    
+    #name {
+      flex: 1;
+      font-size: x-large;
     }
 
     #map {
       height: 100%;
       width: 100%;
     }
+    
+    #notif {
+      z-index: 99999999;
+      border-radius: 25px;
+      padding: 5px;
+      padding-left: 10px;
+      padding-right: 10px;
+      overflow-wrap: break-word;
+      box-shadow: 0 3px 10px 0 #00000030;
+      visibility: hidden; 
+      background-color: #FFF;
+      color: #449f33;
+      position:absolute;
+      transform: translateX(-50%);
+      left:50%;
+      top:10%;
+    }    
+          
+    #notif.show {
+      visibility: visible;
+      animation: fadein 0.5s, fadeout 0.5s 1s;
+    }
+    
+    @keyframes fadein {
+      from {opacity: 0;}
```

Comment:
nit: insert spaces from left and right here, below and above in `position: absolute;` and other properties.

Reply:
fixed 👍